tests/test_monitor_helpers.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+import pandas as pd
+
+from datetime import datetime
+from cosmo.monitor_helpers import convert_day_of_year, fit_line, explode_df, AbsoluteTime
+
+
+class TestConvertDayofYear:
+
+    def test_mjd(self):
+        test_date = 2019.100
+        test_mjd = 58484.0
+
+        result = convert_day_of_year(test_date, mjd=True)
```

Comment:
I'm not sure why this is being tested, ``convert_day_of_year`` seems like it shouldn't exist at all.

Reply:
?

Reply:
It's a wafer-thin wrapper, right, why wrap at all?